### PR TITLE
Fix error with Redshift cluster_id in profiles.yml

### PR DESCRIPTION
**Pull request by jaypeedevlin**
## Description & motivation
Per [thread in slack](https://getdbt.slack.com/archives/CBSQTAPLG/p1636036555299000) this seems to be an error in the documentation.  [This code](https://github.com/dbt-labs/dbt-redshift/blob/465a2682203f8e44f5634e201cc59fcacf462520/tests/unit/test_redshift_adapter.py#L101) seems to support that finding.

### CODE REVIEW

File: website/docs/reference/warehouse-profiles/redshift-profile.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ my-redshift-db:
     dev:
       type: redshift
       method: iam
-      cluster_id: [cluster_id]
+      cluster_id: cluster_id
```
Comment (runleonarun) [inline website/docs/reference/warehouse-profiles/redshift-profile.md]:
```suggestion
      cluster_id: CLUSTER_ID
```

File: website/docs/reference/warehouse-profiles/redshift-profile.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ my-redshift-db:
     dev:
```
Comment (runleonarun) [inline website/docs/reference/warehouse-profiles/redshift-profile.md:62]: I also suggest we add something like this before the example profile:

> Set up your Redshift profile, replacing CLUSTER_ID with the unique identifier of the cluster in Redshift:

File: website/docs/reference/warehouse-profiles/redshift-profile.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ my-redshift-db:
...
       type: redshift
       method: iam
-      cluster_id: [cluster_id]
+      cluster_id: cluster_id
       host: hostname.region.redshift.amazonaws.com
       user: alice
       iam_profile: data_engineer # optional
```
Comment (runleonarun) [inline website/docs/reference/warehouse-profiles/redshift-profile.md:68]: Below here, I also see 
>   db_groups: ['analysts']  

Should [`analysts`] become `ANALYSTS`  if we're standardizing on all caps for text that requires user input?

**Review by runleonarun — DISMISSED** [review]
Hi @jaypeedevlin!

You change is much appreciated! I would like the dbt docs to standardize on all caps for code that requires the user to replace it. I'd also like to see a helpful explanation before the code snippet. I added my 2 cents, but please let me know if I should carry this over the finish line! 🙇🏻‍♀️

**Comment by jaypeedevlin** [comment]
Hi @runleonarun, I'm totally onboard with these changes but unfortunately strapped for time at this busy time of year.  I would love it if you could bring this one home!

**Review by mirnawong1 — APPROVED** [review]

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
thanks @jaypeedevlin and @runleonarun for flagging this and for your patience! gone ahead and merged this for you!

**Pull request merged**